--- a/config.def.h	Thu Apr 30 20:45:50 2026
+++ b/config.h	Thu Apr 30 20:47:40 2026
@@ -92,8 +92,8 @@
 static const char *screenshotcmd[]  = { "xshot.sh", NULL };
 static const char *toggledpmscmd[]  = { "toggle-dpms", NULL };
 static const char *dpmsoffcmd[] = { "xset", "dpms", "force", "off", NULL };
-static const char *upbrightness[]   = { "brightnessctl", "set", "+5%", NULL };
-static const char *downbrightness[] = { "brightnessctl", "set", "5%-", NULL };
+static const char *upbrightness[]   = { "xbacklight", "+", "5", NULL };
+static const char *downbrightness[] = { "xbacklight", "-", "5", NULL };
 static const char *mutecmd[]   = { "pactl", "set-sink-mute", "@DEFAULT_SINK@", "toggle", NULL };
 static const char *volupcmd[]   = { "pactl", "set-sink-volume", "@DEFAULT_SINK@", "+5%", NULL };
 static const char *voldowncmd[] = { "pactl", "set-sink-volume", "@DEFAULT_SINK@", "-5%", NULL };
